From d5e34d5d6f8901a55af41b51f9be61794a9dccf7 Mon Sep 17 00:00:00 2001
From: ahcorde <ahcorde@gmail.com>
Date: Fri, 25 Feb 2022 13:16:48 +0100
Subject: [PATCH] fixed cmakelists

Signed-off-by: ahcorde <ahcorde@gmail.com>
---
 python/CMakeLists.txt | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index bc98f79ce..bd08e206d 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -28,13 +28,11 @@ else()
   set(IGN_PYTHON_INSTALL_PATH ${IGN_LIB_INSTALL_DIR}/python)
 endif()
 
-set(IGN_PYTHON_INSTALL_PATH "${IGN_PYTHON_INSTALL_PATH}/ignition")
-
 # Set the build location and install location for a CPython extension
 function(configure_build_install_location _library_name)
   # Install library for actual use
   install(TARGETS ${_library_name}
-    DESTINATION "${IGN_PYTHON_INSTALL_PATH}/"
+    DESTINATION "${IGN_PYTHON_INSTALL_PATH}/ignition"
   )
 endfunction()
 
@@ -77,7 +75,7 @@ target_link_libraries(sdformat PRIVATE
 )
 
 install(TARGETS sdformat
-	DESTINATION "${IGN_LIB_INSTALL_DIR}/python/"
+	DESTINATION "${IGN_PYTHON_INSTALL_PATH}"
 )
 
 configure_build_install_location(gazebo)
